tests/test_segments.py

```diff
@@ -943,6 +974,199 @@ def test_mode(self):
 		self.assertEqual(i3wm.mode(pl=pl, segment_info={'mode': 'default'}, names={'default': 'test'}), 'test')
 		self.assertEqual(i3wm.mode(pl=pl, segment_info={'mode': 'test'}, names={'default': 'test', 'test': 't'}), 't')
 
+	def test_listers(self):
```

Comment:
This is better to be placed into its own file, this file is named “test_*segments*”.